2024-04-24 09:05:17,504 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:06:17,624 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:07:17,679 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:08:17,774 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:09:17,865 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:10:17,982 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:11:18,092 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:12:18,195 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:13:18,304 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:14:18,388 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:15:18,494 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:16:18,580 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:17:18,692 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:18:18,784 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:19:18,847 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:20:18,925 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:21:18,999 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:22:19,035 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:23:19,154 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:24:19,187 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:25:19,303 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:26:19,422 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:27:19,515 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:28:19,612 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:29:19,724 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:30:19,839 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:31:19,948 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:32:20,052 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:34:52,039 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:35:52,173 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:36:52,293 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:37:52,588 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:38:52,734 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:39:52,819 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:40:52,952 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:41:53,081 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:42:53,216 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:43:53,353 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:44:53,514 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:45:53,593 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:46:53,682 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:47:53,737 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:48:53,845 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:49:53,900 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:50:53,991 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:51:54,105 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:52:54,209 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:53:54,324 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:54:54,439 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:55:54,561 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:56:54,687 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:57:54,712 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:58:54,859 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 09:59:54,980 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:00:55,043 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:01:55,240 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:02:55,362 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:03:55,476 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:04:55,571 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:05:55,686 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:06:55,751 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:07:55,839 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:08:55,921 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:09:56,035 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:10:56,108 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:11:56,125 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:12:56,190 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:13:56,280 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:14:56,389 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:15:56,459 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:16:56,510 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:17:56,566 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:18:56,623 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

2024-04-24 10:19:56,732 ERROR scheduler Exception in Enqueue Events for Site erpnext14.com
Traceback (most recent call last):
  File "apps/frappe/frappe/utils/scheduler.py", line 76, in enqueue_events_for_site
    enqueue_events(site=site)
  File "apps/frappe/frappe/utils/scheduler.py", line 94, in enqueue_events
    queued_jobs = get_jobs(site=site, key="job_type").get(site) or []
  File "apps/frappe/frappe/utils/background_jobs.py", line 298, in get_jobs
    jobs = q.jobs + get_running_jobs_in_queue(q)
  File "apps/frappe/frappe/utils/background_jobs.py", line 335, in get_running_jobs_in_queue
    workers = get_workers(queue)
  File "apps/frappe/frappe/utils/background_jobs.py", line 327, in get_workers
    return Worker.all(queue=queue)
  File "env/lib/python3.10/site-packages/rq/worker.py", line 257, in all
    workers = [
  File "env/lib/python3.10/site-packages/rq/worker.py", line 258, in <listcomp>
    cls.find_by_key(
  File "env/lib/python3.10/site-packages/rq/worker.py", line 685, in find_by_key
    worker.refresh()
  File "env/lib/python3.10/site-packages/rq/worker.py", line 1041, in refresh
    queues = as_text(queues)
  File "env/lib/python3.10/site-packages/rq/utils.py", line 58, in as_text
    raise ValueError('Unknown type %r' % type(v))
ValueError: Unknown type <class 'NoneType'>

